examples/generate_ast.py

```diff
@@ -0,0 +1,44 @@
+# -----------------------------------------------------------------
+# pycparser: generate_ast.py
+#
+# Tiny example of writing an AST from scratch to C code.
+#
+# Andre Ribeiro [https://github.com/Andree37]
+# License: BSD
+# -----------------------------------------------------------------
+from __future__ import print_function
```

Comment:
This is unnecessary now -- I also just removed it from other examples for consistency